src/RemediationRoleSaver.cpp

```diff
@@ -53,22 +82,65 @@ void RemediationSaverBase::selectFilenameAndSaveRole()
 
     try {
         saveToFile(filename);
-        // TODO: if OK - inform the user
+        saveFileOK(filename);
     }
     catch (std::exception& exc)
     {
-        // TODO: if not OK - show error message
-        std::cerr << QObject::tr("Error saving remediation role: %1\n").arg(exc.what()).toUtf8().constData();
+        saveFileError(filename, QString::fromUtf8(exc.what()));
     }
 }
 
-void RemediationSaverBase::saveToFile(const QString& filename)
+
+void RemediationSaverBase::removeFileWhenEmpty(const QString& filename)
+{
+    QFile outputFile(filename);
+    outputFile.open(QIODevice::ReadOnly);
+    int fileSize = outputFile.size();
+    outputFile.close();
+
+    if (fileSize == 0)
+        QFile::remove(filename);
+}
+
+
+void RemediationSaverBase::saveFileOK(const QString& filename)
+{
+    // TODO: if OK - inform the user
+}
+
+
+void RemediationSaverBase::saveFileError(const QString& filename, const QString& errorMsg)
```

Comment:
why are there 2 newlines between methods everywhere? shouldn't it be 2 lines between different classes but 1 line between methods?